x/protorev/types/keys.go

```diff
@@ -97,6 +98,9 @@ var (
 
 	// KeyCyclicArbTracker is the prefix for store that keeps track of the height we began tracking cyclic arbitrage
 	KeyCyclicArbTrackerStartHeight = []byte{prefixcyclicArbTrackerStartHeight}
+
+	// KeyPrefixBaseDenoms is the prefix that is used to store the base denoms that are used to create cyclic arbitrage routes
+	KeyPrefixBaseDenoms = []byte{prefixBaseDenoms}
```

Comment:
I could have probably just used the same old key, but imo after dealing with some of the tests (specifically the upgrade tests) it seemed prudent to just create a new key.

Reply:
SGTM